src/yokozuna_rt.erl

```diff
@@ -56,6 +58,14 @@
 host_entries(ClusterConnInfo) ->
     [riak_http(I) || {_,I} <- ClusterConnInfo].
 
+%% @doc Generate `SeqMax' keys. Yokozuna supports only UTF-8 compatible keys.
+-spec gen_keys(pos_integer()) -> list().
```

Comment:
I'm not an expert on xref, but are we allowed to say the return value is a list of binary?  [binary()], maybe?